docs/src/api/class-cdpsession.md

```diff
@@ -45,6 +45,16 @@ client.send("Animation.setPlaybackRate", {
     playbackRate: response["playbackRate"] / 2
 })
 ```
+```csharp
+var client = await Page.Context.NewCDPSessionAsync(Page);
```

Comment:
this example is not working. 

line 51: maybe we need an overload for callbacks without arguments.
line 54/55/56 should be `playbackRate` instead of `response.playbackRate`.

Reply:
Adding an overload for callbacks without arguments would mean maintaining 2 dictionaries of callbacks, one for Action<JsonElement> and one for Action. This might be a nice to have in the future, but for now I'm just going to use a discard for the unwanted arguments.